doc/python/server_reflection.md

```diff
@@ -49,6 +49,61 @@ is working properly by using the [`grpc_cli` command line tool]:
   please refer to the [`grpc_cli` documentation] and the
```

Comment:
Can you also update the Sphinx rst under doc/python to include this API?

Reply:
Done (actually started working on it when I pushed the previous commit, but it took a while).